.github/workflows/continuous-integration.yml

```diff
@@ -17,27 +17,23 @@ jobs:
         distribution: 'temurin'
         cache: maven
     - name: Checkout wala/IDE sources.
-      uses: actions/checkout@v3.5.2
-      with:
-        repository: wala/IDE
-        # fetch-depth: 50
-        path: ./IDE
+      run: git clone --depth=50 https://github.com/wala/IDE ${{ runner.temp }}/IDE
```

Comment:
What is the purpose of these changes to use an explicit `git clone`?  I don't quite get what is going on here.

Reply:
Sorry for the lack of context. The XML format checker is going into the Jython3 code that is being cloned inside the project directory and causing a failure. The idea then is to move the cloned external dependencies outside the project directory, as was done in the original Travis CI configuration. However, the GitHub Actions only allows cloning in the project directory. Thus, we reverted to the explicit clone, as was also done in the original Travis CI configuration.

Reply:
Relevant lines from Travis CI config:

https://github.com/wala/ML/blob/6c10952c9ca09afc2846374555251b2e7b0966e8/.travis.yml#L6-L7

Reply:
Does it not work to keep using the checkout action but pass `path: ${{ runner.temp }}/IDE` instead of `path: ./IDE`?

Reply:
It does not, unfortunately. That variable returns a path outside the project directory but in the working space.

Reply:
https://github.com/wala/ML/pull/61/commits/39ac40e41630fb65c04013bca1bcedeea385ace1

Reply:
Ok, yes, this is indeed a limitation, sigh https://github.com/actions/checkout/issues/197